### ISSUE TITLE
Panel question

### ISSUE THREAD

**Issue by Nobellium1** [UnamSanctam/SilentCryptoMiner#692]
Hello.

I'd like to ask one thing about the panel.

I'd like to use multiple addresses on the same panel, and not only one. Am I able to completely remove configurations so it never uses configs from panel but only from rentry because like this it moves em all on 1 address that is saved in ethminer config

**Comment by UnamSanctam** [comment]
It will only use a configuration from the panel if you enter one into it, if you leave all configurations empty (change to `{}` if you already have things there) then it won't apply any configuration.

**Comment by Nobellium1** [comment]
Aight thanks a lot

**Comment by Nobellium1** [comment]
Is there a way to remove someone from panel?

**Comment by Nobellium1** [comment]
Like person that was offline for so long so I just dont wanna see him on the panel anymore

**Comment by UnamSanctam** [comment]
Yes the "Remove Miner" button (after pressing the '+' button next to the miner ID to see everything) removes them from the database.

**Comment by Nobellium1** [comment]
Perfect, ty. Also what do you think it would be if I would crypt the downloader (with like 3 different files to be downloaded - that are NOT crypted) and set a delay of like 40-50 seconds. And have WD exclusions ON. Do you think WD would still detect them or it would first add exclusions and then download them so they never get detected?

**Comment by UnamSanctam** [comment]
The delay shouldn't matter, it will first add the Windows Defender exclusions and then after they have been added should the files be downloaded.

**Comment by Nobellium1** [comment]
But the last time I did it like that, it still got detected (files inside it) even the downloader was fully crypted

**Comment by Nobellium1** [comment]
What I think is that it didnt add right exclusions so those files werent excluded from WD, don't see the other reason that would happen

**Comment by UnamSanctam** [comment]
Did it detect the downloaded files themselves after being downloaded or something different?

**Comment by Nobellium1** [comment]
I cant remember but like 99% sure it was files

**Comment by UnamSanctam** [comment]
Maybe it didn't wait for the powershell command (that adds exclusions) to finish but instead maybe only started it externally before moving on the next command. I'm currently in the process of updating the downloader and binder so maybe you can see if that happens whenever I release them.

**Comment by Nobellium1** [comment]
Perfect, cant wait to test it out
